.github/workflows/ci.yml

```diff
@@ -112,8 +115,8 @@ jobs:
   # Deployment steps taken from https://github.com/colinwilson/static-site-to-vercel/blob/master/.github/workflows/deploy-preview.yml
   repl_build:
     name: Build REPL
+    if: ${{ github.event_name == 'pull_request' }}
     runs-on: ubuntu-latest
-    needs: [unit_tests]
```

Comment:
Was this due a dependency on unit tests, or to avoid building the REPL if the unit tests weren't going to pass? I know you mentioned it re-uses the rust binaries, but was that just an optimisation since this build waited for unit tests to pass anyway?

Does the deploy for the REPL only happen if the other tests pass?

Reply:
I'll need @devongovett to confirm the original intention, but assumed it's not needed since that's how the release flow works now

Reply:
> I know you mentioned it re-uses the rust binaries, but was that just an optimisation since this build waited

Yes, that was the idea when I added the REPL to CI. I don't think the REPL for PRs should only be built if all tests pass